lib/net.js

```diff
@@ -719,27 +739,31 @@ function createWriteReq(req, handle, data, encoding) {
 }
 
 
-Socket.prototype.__defineGetter__('bytesWritten', function() {
-  var bytes = this._bytesDispatched,
-      state = this._writableState,
-      data = this._pendingData,
-      encoding = this._pendingEncoding;
-
-  state.getBuffer().forEach(function(el) {
-    if (el.chunk instanceof Buffer)
-      bytes += el.chunk.length;
-    else
-      bytes += Buffer.byteLength(el.chunk, el.encoding);
-  });
+Object.defineProperty(Socket.prototype, 'bytesWritten', {
+  get: function() {
```

Comment:
Enhanced Object Literals (e.g. `get() {`) Just bringing it up now b/c I'm sure someone will. I don't care though.